### DISCUSSION TITLE
Do margins factors (MEFs) include transport to final consumer?

### DISCUSSION THREAD

**Discussion by MoLi7** [USEPA/supply-chain-factors#9] [Q&A] [answered]
As transport is included in margins.

**Comment by MoLi7** [comment, marked as answer]
If delivery to the final user is provided by the good or service provider, then YES.

If the consumer provides the transport, then NO.​

**Reply by Brian-Solomon** [reply]
@WesIngwersen - this answer seems incomplete based on direct discussions we have had via email. We discussed that the majority of emissions represented by MEFs pertain to transportation/distribution between a direct manufacturer and a retailer/wholesaler (but _may_ include some _shelf-to-buyer_ emissions). We have been including MEFs for all purchases made via retailer/wholesaler entities and excluding them only for direct-from-manufacturer purchases. For Scope 3 Category 4 emissions, we've been working with organizations to separate out any direct spend on shipping/freight of purchased goods, as this would fall within a _shelf-to-buyer_ segment of the supply chain (i.e., **_after_** the point of sale).

![image](https://user-images.githubusercontent.com/101364203/206573428-5f3d2bdf-9ebf-461f-b068-36b01a93342c.png)
